src/translations/en.json

```diff
@@ -936,6 +936,19 @@
             "required_fields": "Fill in all required fields"
           }
         }
+      },
+      "lovelace": {
```

Comment:
There already is a lovelace category, you should merge your changes into that one.